<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="sandpit.UAT_BavarianInnBrokenLinks" tests="8" time="0.016" ignored="0" failures="0" timestamp="17 Jul 2019 15:23:31 GMT" skipped="7" hostname="DESKTOP-G1T6KU3" errors="1">
  <testcase name="titleTest" time="0.002" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- titleTest -->
  <testcase name="membersSocial" time="0.003" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- membersSocial -->
  <testcase name="loginToSite" time="0.001" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- loginToSite -->
  <testcase name="membersOffers" time="0.007" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- membersOffers -->
  <testcase name="membersLocations" time="0.001" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- membersLocations -->
  <testcase name="WelcomeMessage" time="0.001" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- WelcomeMessage -->
  <testcase name="setUpTest" time="0.000" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at sandpit.UAT_BavarianInnBrokenLinks.setUpTest(UAT_BavarianInnBrokenLinks.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.TestRunner.beforeRun(TestRunner.java:634)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUpTest -->
  <testcase name="logOutOfSite" time="0.001" classname="sandpit.UAT_BavarianInnBrokenLinks">
    <skipped/>
  </testcase> <!-- logOutOfSite -->
</testsuite> <!-- sandpit.UAT_BavarianInnBrokenLinks -->
